### ISSUE TITLE
Damage-Taken statistic is sometimes super inflated

### ISSUE THREAD

**Issue by Xerocoles** [PGMDev/PGM#953]
Heyo, so here's a super annoying stat issue that affects both Bolt & Stratus.  As someone who's looked through countless Stratus match links of player stats, and had to use them to make tons of calculations, I can say this is 100% happening at times (I'll prove it in a sec) yet it's only very rarely.  The vast majority of times, all the statistics seem perfectly fine.  Yet every now and then you'll see one that seems suspicious in the Damage-Taken column only, or one where Damage-Taken is just completely obviously way off.   Obviously it'll be slightly higher in total than the total Damage-Dealt since there's fall damage and void damage, but that's not what this is about.

Example 1 (https://stratus.network/match/7457043571461)
![Screen Shot 2022-01-09 at 10 28 35 PM](https://user-images.githubusercontent.com/54342797/148715311-91694e45-ee87-4767-9f0f-7f1c418f9467.png)
The damage dealt by red team is 1069.  The damage taken by blue team is 1726.  That means that for this to be true, Blue had to take almost 700 hearts in fall damage and/or void damage on Coven.  Well that's just not true, I watched the whole game.  If you want some reference, looking at the non-bugged stats of red team, they took a total of 26.5 hearts extra damage, not anything close to 700.  So in all likelihood, Blue is also probably in the 20-30 range.  I used to think it was commonly just multiplied by x2 perfectly, but these all seem bugged, and if you cut them all in half, you end up way short of the damage-dealt, which is impossible.  I tried removing Mamut's cut, since maybe his is legit, but it's still not enough.  So clearly it's not JUST a "sometimes becomes x2" issue, even though that often seems reasonable.

Example 2 (https://bolt.rip/matches/d719e2)
![Screen Shot 2022-01-09 at 10 34 27 PM](https://user-images.githubusercontent.com/54342797/148715723-27a25856-7272-48db-99d4-414a62d73db3.png)
This is basically just to show that it's also happened on Bolt.  Guapote, Hielos, FeyRune, PepeLMAO, Zeus, and TOOOR all appear to be bugged, some of them definitely, (Guapote) and some just probably.

Example 3
<img width="986" alt="Screen Shot 2022-01-09 at 10 42 11 PM" src="https://user-images.githubusercontent.com/54342797/148716072-b1a4634e-ab7d-4d7f-9066-49b534374521.png">
This one is slightly older, not sure the match it's from.  Unless it was from somehow taking billions of fall damage hearts like from a staff doing /rocket, it just doesn't make any sense.  And I seriously doubt that's the case, the only reason it's a chance is because I don't know the context on this one lol.  But anyways, it mainly proves that it can be bugged in many ways, like this one is WAY higher than it should be.

Extra sidenotes:
- I've wondered if it's something map-related, like taking some sort of "phantom fall damage" in spawntube drops, or anything idk, but there never seems to be a map or gamemode pattern as far as I can tell.
- It's not just a website issue on both servers, cause the papers in-game are always bugged in the same way when this happens.
- In the Stratus example you can see it affected all of the people on one team, and zero people on the other team.  Seems patterned, and I think a pattern like that is generally kinda common.  But then not always?  Like the bolt one, it has some people on both teams, yet not either full team.  Weird.

**Comment by Pablete1234** [comment]
This is a bit too much text imo,

TLDR is that the damage taken stat tracked by pgm is sometimes higher than it should, this should be visible in-game on the paper with stats if that's the case so checking websites shouldn't be needed, and you have confirmed that it does happen

Do we know under which conditions this happens?

**Comment by Xerocoles** [comment]
We have found one clue finally! -- you can apply damage to players while being a spectator, i.e. pulling out a diamond sword and critting someone

**Comment by KingOfSquares** [comment]
#957
